### ISSUE TITLE
 High processor and memory consumption

### ISSUE THREAD

**Issue by coutinho2010** [Looking4Group/L4G_Core#3645]
High processor and memory consumption with only 10 players online server xeon 3.0ghz + 6gb ram

**Comment by Saltgurka** [comment]
Perhaps you have this enabled in your config file?
```

#    LoadAllGridsOnMaps
#        Load grids of maps at server startup (if you have lot memory you can try it to have a living world always loaded)
#        This also allow ALL creatures on the given maps to update their grid without any player around.
#        Default: "" (don't load all grids at startup)
#                 "mapId1[,mapId2[..]]" (DO load all grids on the given maps- Experimental and very resource consumming)
```

**Comment by coutinho2010** [comment]
Is in "default" mode even so when I reboot the server the processor stays at 0% and normal memory consumption as the players go on the map will increase the consumption of both.

**Comment by coutinho2010** [comment]
Would not be able to disable this loadallgridonmaps?

**Comment by Xadras** [comment]
Is GridUnload in conf set to default value? --> 1

**Comment by coutinho2010** [comment]
Yes it is all default I use windows server 2008 r2 the core conf config I did not change anything what happens is that as players walk on the map the processor goes up and the memory consumption increasing even all the players leaving continues the consumption of both.

**Comment by AnonXS** [comment]
Well there are alot of ways to reduce processor load for the core, most easily achieved by changing the server config file, if you find any other ways of improving this state report back.

**Comment by Rapth** [comment]
is this problem solved? friend?
